frontend/src/components/Panel.svelte

```diff
@@ -9,7 +9,7 @@
 
     //Boolean variables
     export let show_legend_placeholder: boolean; 
-    export let show_menu_placeholder: boolean; 
+    /* export let show_settings_placeholder: boolean; */ 
```

Comment:
Any particular reason to leave this here?

Reply:
No, forgot to remove! Will fix it!